tests/io/dask/dataframe/test_repartition.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import pytest
+
+from kartothek.core.factory import DatasetFactory  # noqa: E402
+from kartothek.io.dask.dataframe import read_dataset_as_ddf
+from kartothek.io.dask.repartition import (
+    calculate_repartition_ratio,
+    optimize,
+    repartition_ddf,
+)
+from kartothek.io.iter import store_dataframes_as_dataset__iter
+from kartothek.io_components.metapartition import SINGLE_TABLE
+
+
+@pytest.fixture
+def dataset_factory_function_scope(store_factory):
+    return DatasetFactory(dataset_uuid="test", store_factory=store_factory)
+
+
+@pytest.mark.parametrize("frac", (1.0, 0.5, 0.1))
```

Comment:
Use either 0.5 or 0.1. There is not much need for both since both are partial fractions. With the additional parametrization below we should keep combinatorics in check.

Reply:
using 0.5 . Removed 0.1